discord/client.py

```diff
@@ -644,8 +644,25 @@ async def start(self, token: str, *, reconnect: bool = True) -> None:
             An unexpected keyword argument was received.
         """
         await self.login(token)
+        await self.setup()
         await self.connect(reconnect=reconnect)
 
+    async def setup(self) -> Any:
```

Comment:
```suggestion
    async def setup_hook(self) -> None:
```

Reply:
Why shouldn't this return Any, the return from `setup_hook` isn't used so shouldn't be enforced to be a specific type imo

Reply:
Precisely because it's unused.